CONTRIBUTING.md

```diff
@@ -25,7 +25,8 @@ make
 
 ### Running tests
 
-This project contains unit tests and integration tests with coverage collection. See [tests/README.md](./tests/README.md) for how to execute and add tests.
+This project contains unit tests and integration tests with coverage collection.
```

Comment:
Could u change the "Format of the Commit Message" in this pr?  
From:
```
<subsystem>: <what changed>
```
To:
```
<subsystem>(ticdc): <what changed>
```

Reply:
See: https://github.com/pingcap/ticdc/pull/3549